### PR TITLE
Arome/ Remove Step fx from forex

### PR DESCRIPTION
**Pull request by arome-binary**
Changes:

- remove step-fx 

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [x] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by suthesan-deriv — APPROVED** [review]

**Review by yashim-deriv — APPROVED** [review]

File: src/pages/markets/static/content/_margin.js [hunk 1]
```diff
@@ -12,14 +12,13 @@ import {
...
 } from '../../instruments/_submarkets'
-import { smart_gold_index, step_fx } from '../../instruments/_market-symbols'
+import { smart_gold_index } from '../../instruments/_market-symbols'
 import {
     CrashBoomDetails,
     EuropeDetails,
     RangeBreakIndicesDetails,
     SmartFXDetails,
     SmartGoldIndexDetails,
-    StepFXDetails,
```
Comment (sean-binary) [inline src/pages/markets/static/content/_margin.js:22]: I think you may have forgotten to remove `StepFXDetails` from `markets/static/content/_details.js` as well.

Comment (sean-binary) [inline src/pages/markets/static/content/_margin.js:22]: I think you may have forgotten to remove `StepFXDetails` from `markets/static/content/_details.js` as well.

**Review by sean-binary — CHANGES_REQUESTED** [review]

File: src/pages/markets/static/content/_margin.js (hunk 1 above)
Comment (arome-binary) [reply src/pages/markets/static/content/_margin.js:22]: That has already been removed; it is already highlighted in red bg.

Comment (seanho96) [reply src/pages/markets/static/content/_margin.js:22]: Nono, I am referring to the one in the `_details.js` file.

Comment (sean-binary) [reply src/pages/markets/static/content/_margin.js:22]: Nono, I am referring to the one in the `_details.js` file.

Comment (arome-binary) [reply src/pages/markets/static/content/_margin.js:22]: Gotcha! on it!

Comment (arome-binary) [reply src/pages/markets/static/content/_margin.js:22]: Updated on this PR, kindly check it out, thanks.... https://github.com/binary-com/deriv-com/pull/1567/files

**Pull request closed without merging**